src/components/Modals/index.tsx

```diff
@@ -38,9 +38,15 @@ export const TransactionModal: React.SFC<TransactionModalProps> = ({
       <ul>
         <li>{`Sell Amount:    ${txData.sellAmount}`}</li>
         {(txData.tokenA.symbol || txData.tokenA.name) &&
-        <li>{`Token Selling:  ${txData.tokenA.symbol || txData.tokenA.name} [${txData.tokenA.name || txData.tokenA.symbol}]`}</li>
+        <li>{`Token Selling:  ${txData.tokenA.symbol || txData.tokenA.name}${txData.tokenA.name && ' [' + txData.tokenA.name + ']'}`}</li>
```

Comment:
If a token only has `.name` but not `.symbol` this will be `Token Selling: Ethereum Token[Ethereum Token]`
Maybe go with `${txData.tokenA.name && !txData.tokenA.symbol  && ' [' + txData.tokenA.name + ']'}` for the last part.

Reply:
👍

Reply:
Yep, you're right, I was thinking of something else
<sup>flip a variable</sup> <small>flip a variable</small> <sub>flip a variable</sub>